src/AltinnCore/ServiceLibrary/Configuration/ServiceConfiguration.cs

```diff
@@ -8,19 +8,19 @@ namespace AltinnCore.ServiceLibrary.Configuration
     public class ServiceConfiguration
     {
         /// <summary>
-        /// Gets or sets the service code
+        /// Gets or sets the repository name
         /// </summary>
-        [RegularExpression("^[a-zA-Z][a-zA-Z0-9_\\-]{2,30}$", ErrorMessage = "Må begynne med en bokstav og ikke inneholde mellomrom eller spesialtegn ('-' er tillatt)")]
-        public string Code { get; set; }
+        [RegularExpression("^[a-zA-Z]+[a-zA-Z0-9_]*$", ErrorMessage = "Må begynne med en bokstav og ikke inneholde mellomrom eller spesialtegn ('_' er tillatt)")]
+        public string RepoName { get; set; }
```

Comment:
This can also be named as "RepositoryName" to be consistent with the naming in service meta data